### DISCUSSION TITLE
How to change the connected Hyundai account

### DISCUSSION THREAD

**Discussion by 4homeassistant** [Hyundai-Kia-Connect/kia_uvo#512] [General]
I just wanted to connect a different Hyundai account but unfortunately I cannot find a way to do that. The only option I found seems to be dead. If I click on this "Connected Devices" nothing happens. Any idea?
![image](https://user-images.githubusercontent.com/117555636/208418904-c0dc61c3-1408-4a8e-9ebe-dd83acae0f77.png)

**Comment by cdnninja** [comment]
Add the integration a second time and it should allow you to set it up again assuming email is different.

**Comment by 4homeassistant** [comment]
@cdnninja , Thank you! Worked well. Both cars show up now. What about adding this to the documentation? By the way - do you guys need translation support? I could help with the German translation but have no glue, how to deal with Github so far.

**Reply by cdnninja** [reply]
Yes and yes!  For docs are you able to hit edit on the read me on the home page and propose changes? 

As for translations always appreciated! A folder exists in the home page for translations where a new file would be created that would contain it.

**Reply by 4homeassistant** [reply]
Re docs, is it "Fork this repository and edit the file" when selecting README.md?

**Reply by 4homeassistant** [reply]
Re translation, I found a file 'en.json' in /translations that just contains a few items regarding initial setup. Any chance to get the entities translated? Already did that in my HA.

**Reply by 4homeassistant** [reply]
Tried it but have no idea if it's been the correct way. Happy to get feedback 😉

**Reply by cdnninja** [reply]
> Re docs, is it "Fork this repository and edit the file" when selecting README.md?

Yes.  You edit and then create a pull request to update our repo.  

That's correct for the translation.  I'll take a look to see how we can do it for all sensors.  This is a good start though! I haven't looked as sensors in the last since haven't had someone to translate! Thank you!
